pkg/loghttp/legacy/query.go

```diff
@@ -11,12 +11,6 @@ type QueryResponse struct {
 
 // Stream represents a log stream.  It includes a set of log entries and their labels.
 type Stream struct {
```

Comment:
I think the `Stream` struct's definition should be moved to the model package.

Reply:
@owen-d This pkg also have some issue. It is using `"github.com/prometheus/common/model"` and exporting `Vector and Matrix`